locust-llm-benchmark/benchmark.py

```diff
@@ -0,0 +1,84 @@
+import json
+import os
+import time
+
+from locust import HttpUser, between, events, task
+
+PROMPT = "Explain MLOps in 300 tokens or less."
+TFY_HOST = os.getenv("TFY_HOST", "https://llm-gateway.truefoundry.com")
+# or export self hosted url Ex: https://*.truefoundry.tech/
+
+TFY_API_KEY = os.environ["TFY_API_KEY"]
```

Comment:
Rename env var to 

OPENAI_BASE_URL (send req to OPENAI_BASE_URL/chat/completions or /completions)
OPENAI_API_KEY

take these two env var as input.

Reply:
actually we can skip the OPENAI_BASE_URL because locust me that is required as an input